### PR TITLE
Fix #279: Added PR close step to CLA Check Github Action

### LINKED ISSUE #279: Github action bot should close the PR if the author has not signed the CLA
OppiaBot use to close the PR if the PR author has not signed the CLA. We migrated the CLA check to the Github-actions as the OppiaBot feature was not reliable. 

Currently, the github-action bot is not closing the PR in such a situation. Through this issue, we want to ensure that OppiaBot closes such PRs.

### PR DESCRIPTION
**Pull request by gp201**
## Explanation

This PR fixes #279
Now the PR is closed when the LCA check fails and puts a comment  to guide the contributor on further steps

## Checklist
- [ ] I have successfully deployed my own instance of Oppiabot.
  - You can find instructions for doing this [here](https://github.com/oppia/oppiabot/wiki/Deploying-your-own-instance-of-the-oppiabot).
- [ ] I have manually tested all the changes made in this PR following the [manual tests matrix](https://github.com/oppia/oppiabot/wiki/Manual-Tests-Matrix).

### CODE REVIEW

**Comment by gp201** [comment]
@vojtechjelinek @jameesjohn PTAL

File: actions/src/dispatcher.js [hunk 1]
```diff
@@ -31,8 +31,10 @@ module.exports = {
     const repoName = context.payload.repository.name.toLowerCase();
     const checksWhitelist = constants.getChecksWhitelist();
     if (Object.prototype.hasOwnProperty.call(checksWhitelist, repoName)) {
+      core.info('Repo is whitelisted');
```
Comment (vojtechjelinek) [inline actions/src/dispatcher.js:34]: Do we need to add this? Ditto below.

File: spec/claCheckSpec.js [hunk 2]
```diff
@@ -186,7 +189,17 @@ describe('CLA check github action Module for Oppia', () => {
         ' Welcome to Oppia! Could you please ' +
         'follow the instructions ' + LINK_RESULT +
         " and sign the CLA Sheet to get started? You'll need to do " +
-        'this before we can accept your PR. Thanks!');
+        "this before we can accept your PR. Once you\'re done," +
+        ' please reopen the PR. Thanks!');
+
+      expect(octokit.issues.update).toHaveBeenCalled();
+
+      expect(octokit.issues.update).toHaveBeenCalledWith({
```
Comment (vojtechjelinek) [inline spec/claCheckSpec.js:197]: I think just `toHaveBeenCalledWith` is enough, since it checks for some call implicitely.

Comment (vojtechjelinek) [reply spec/claCheckSpec.js:197]: ditto below

File: actions/src/pull_requests/claCheck.js [hunk 3]
```diff
@@ -69,9 +69,16 @@ const generateOutput = async (hasClaSigned) => {
         ' Welcome to Oppia! Could you please ' +
         'follow the instructions ' + LINK_RESULT +
         " and sign the CLA Sheet to get started? You'll need to do " +
-        'this before we can accept your PR. Thanks!');
-    core.info('Commenting in PR...');
+        "this before we can accept your PR. Once you\'re done," +
```
Comment (vojtechjelinek) [inline actions/src/pull_requests/claCheck.js:72]:
```suggestion
        "this before we can accept your PR. Once you're done," +
```

**Review by vojtechjelinek — CHANGES_REQUESTED** [review]
Thanks! Left a few comments.

File: actions/src/dispatcher.js (hunk 1 above)
Comment (gp201) [reply actions/src/dispatcher.js:34]: Yes, It makes debugging a bit easier for future developers when introducing new actions.

File: actions/src/pull_requests/claCheck.js (hunk 3 above)
Comment (gp201) [reply actions/src/pull_requests/claCheck.js]: Done

File: spec/claCheckSpec.js (hunk 2 above)
Comment (gp201) [reply spec/claCheckSpec.js]: Done

**Comment by gp201** [comment]
@vojtechjelinek PTAL

File: actions/src/dispatcher.js (hunk 1 above)
Comment (vojtechjelinek) [reply actions/src/dispatcher.js:34]: Ok.

**Review by vojtechjelinek — APPROVED** [review]
LGTM!

**Review by jameesjohn — APPROVED** [review]
LGTM!

**Pull request merged**